sympy/solvers/solvers.py

```diff
@@ -457,102 +471,105 @@ def solve(f, *symbols, **flags):
     r"""
     Algebraically solves equations and systems of equations.
 
-    Currently supported are:
-        - polynomial,
+    Explanation
+    ===========
+
+    Currently supported:
+        - polynomial
         - transcendental
         - piecewise combinations of the above
         - systems of linear and polynomial equations
-        - systems containing relational expressions.
+        - systems containing relational expressions
 
     Input is formed as:
 
-    * f
-        - a single Expr or Poly that must be zero,
+    * *f*
+        - a single Expr or Poly that must be zero
         - an Equality
         - a Relational expression
         - a Boolean
         - iterable of one or more of the above
 
-    * symbols (object(s) to solve for) specified as
+    * *symbols* (object(s) to solve for) specified as
         - none given (other non-numeric objects will be used)
         - single symbol
         - denested list of symbols
-          e.g. solve(f, x, y)
+          (e.g., ``solve(f, x, y)``)
         - ordered iterable of symbols
-          e.g. solve(f, [x, y])
+          (e.g., ``solve(f, [x, y])``)
 
-    * flags
+    * *flags*
         'dict'=True (default is False)
-            return list (perhaps empty) of solution mappings
+            Return list (perhaps empty) of solution mappings.
         'set'=True (default is False)
-            return list of symbols and set of tuple(s) of solution(s)
+            Return list of symbols and set of tuple(s) of solution(s).
         'exclude=[] (default)'
-            don't try to solve for any of the free symbols in exclude;
+            Do not try to solve for any of the free symbols in exclude;
             if expressions are given, the free symbols in them will
             be extracted automatically.
         'check=True (default)'
-            If False, don't do any testing of solutions. This can be
-            useful if one wants to include solutions that make any
+            If False, do not do any testing of solutions. This can be
+            useful if you want to include solutions that make any
             denominator zero.
         'numerical=True (default)'
-            do a fast numerical check if ``f`` has only one symbol.
+            Do a fast numerical check if *f* has only one symbol.
         'minimal=True (default is False)'
-            a very fast, minimal testing.
+            A very fast, minimal testing.
         'warn=True (default is False)'
```

Comment:
I don't know why these flags all use this weird quote formatting. Really all of these should be under the parameters section.

Reply:
Yes this section is formatted strangely. I tried to make it a parameters section at first, but it looked odd because of the bulleted lists under each parameter making the formatting kind of weird. I'll try it as a parameters section again and you can tell me what you think.

I also thought the single quotes around the flags were odd, but have also seen this elsewhere, like under opt_cse, so thought maybe it was similar to that?

def opt_cse(exprs, order='canonical'):
    """
    Find optimization opportunities in Adds, Muls, Pows and negative
    coefficient Muls.

    Parameters
    ==========

    exprs : list of sympy expressions
        The expressions to optimize.
    order : string, 'none' or 'canonical'
        The order by which Mul and Add arguments are processed. For large
        expressions where speed is a concern, use the setting order='none'.

    """

Reply:
In that case, the right hand side is in quotes because it is saying that the `order` parameters should be the string `'none'` or `'canonical'`. But here, it doesn't make sense for the left hand side, the parameter name like `warn` to be a string. `'warn'=True` isn't valid Python. And having the whole thing be a string makes even less sense. I also saw an example of this in the text of another docstring in this file.

Reply:
> I also saw an example of this in the text of another docstring in this file.

It was `'dict'=True` in the `nsolve` docstring. That should just be `dict=True` (formatted as code).

Reply:
Ok I'll make those changes!